addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -610,6 +610,149 @@ QUnit.test('chat window: open / close', async function (assert) {
     );
 });
 
+QUnit.test('chat window (mobile): open channel with no server notification', async function (assert) {
+    assert.expect(2);
+
+    this.data['mail.channel'].records.push({
+        id: 20,
+        uuid: 'channel-uuid',
```

Comment:
No need to add field not used in the test

Reply:
Done